### PR TITLE
CLI Reference - Quick typo/edits

### PR DESCRIPTION
**Pull request by sarah11918**
Just minor fixes, and updated to the same verb tense throughout all the commands.

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
This one looks good too!

File: src/pages/en/reference/cli-reference.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ title: CLI Reference
 
 ### `astro dev`
 
-Runs the Astro development server. This starts an HTTP server that responds to requests for pages stored in `src/pages` (or which folder is specified in your [configuration](/en/reference/configuration-reference)).
+Runs the Astro development server. This starts an HTTP server that responds to requests for pages stored in `src/pages` (or whichever folder is specified in your [configuration](/en/reference/configuration-reference)).
```
Comment (aFuzzyBear) [inline src/pages/en/reference/cli-reference.md:10]:
```suggestion
Runs the Astro development server. This starts an HTTP server that responds to requests for pages stored in `src/pages` (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference)).
```

File: src/pages/en/reference/cli-reference.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Builds your site for production.
```
Comment (aFuzzyBear) [inline src/pages/en/reference/cli-reference.md:27]:
```suggestion
Builds your site for production based upon your [configuration options](/en/reference/configuration-reference)) and is produced to the `./dist`.
```

**Review by aFuzzyBear — COMMENTED** [review]
let me know what you think buddy

File: src/pages/en/reference/cli-reference.md [hunk 3]
```diff
@@ -7,17 +7,17 @@ title: CLI Reference
 
 ### `astro dev`
 
-Runs the Astro development server. This starts an HTTP server that responds to requests for pages stored in `src/pages` (or which folder is specified in your [configuration](/en/reference/configuration-reference)).
+Runs  Astro's `dev` server. It starts a HTTP server which responds to requests for routes or pages that are specified within `src/pages` directory (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference)).
```
Comment (sarah11918) [inline src/pages/en/reference/cli-reference.md:10]:
```suggestion
Runs  Astro's `dev` server. It starts an HTTP server which responds to requests for routes or pages that are specified within `src/pages` directory (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference)).
```

**Comment by FredKSchott** [comment]
LGTM!

**Pull request merged**